src/main/resources/assets/computercraft/lua/rom/help/lsper

```diff
@@ -0,0 +1 @@
+lsper list you all attached peripherals
```

Comment:
Write help files in the passive voice, that applies to the others as well. This just sounds really odd.